must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -113,3 +109,8 @@ echo "collecting dump of oc get pvc all namespaces" | tee -a  "${BASE_COLLECTION
 echo "collecting dump of oc get obc all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
 { oc get obc --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/obc_all_namespaces"
 { oc adm --dest-dir="${BASE_COLLECTION_PATH}/namespaces/all/" inspect obc --all-namespaces --"${SINCE_TIME}"; } >> "${BASE_COLLECTION_PATH}/gather-debug.log" 2>&1
+
+# For VolumeReplication of all namespaces
+echo "collecting dump of oc get volumereplication all namespaces" | tee -a  "${BASE_COLLECTION_PATH}/gather-debug.log"
+{ oc get volumereplication --all-namespaces; } >> "${BASE_COLLECTION_PATH}/namespaces/all/vr_all_namespaces"
```

Comment:
please collect `-oyaml` output also

Reply:
@Madhu-1 the command at line 116 does that. It will collect the yaml of all namespaces at the given location.